application/classes/Ushahidi/Console/User.php

```diff
@@ -0,0 +1,68 @@
+<?php defined('SYSPATH') or die('No direct script access');
+
+/**
+ * Ushahidi User Console Command
+ *
+ * @author     Ushahidi Team <team@ushahidi.com>
+ * @package    Ushahidi\Console
+ * @copyright  2014 Ushahidi
+ * @license    https://www.gnu.org/licenses/agpl-3.0.html GNU Affero General Public License Version 3 (AGPL3)
+ */
+
+use Ushahidi\Console\Command;
+use Ushahidi\Core\Entity\UserRepository;
+use Ushahidi\Core\Tool\Validator;
+use Ushahidi\Core\Exception\ValidatorException;
+
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+
+class Ushahidi_Console_User extends Command {
```

Comment:
Could this live in Ushahidi\Console\User instead? I don't see anything here thats Kohana dependent